pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -128,98 +101,82 @@ social:
-      = hoc_s(:front_intro_default_2)
-      %br
-      = view :hoc_events_counter
-
-  %br/
-  %br/
+  %section.hoc-description
+    .wrapper
+      %h2.number-served=hoc_s(:front_join_us_n_students_served).gsub("#", format_integer_with_commas(fetch_hoc_metrics['started']).to_s)
+      %p=hoc_s(:front_intro_default_2)
+      %p=view :hoc_events_counter
 
   -# Map
-  - unless @country == 'ma' # Hide map for Morocco because of sensitive border disputes misrepresented in map
-    #maparea.full-width
-      -# Show a picture of the map unless it is currently the week of HOC.
-      -if hoc_mode != "actual-hoc"
-        = view :hoc_events_map
-        .footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals: {events_url: resolve_url('/events')})
-      -else
-        %a{href: resolve_url('/map')}
-          %img{src: '/images/fit-2276/map.jpg', style: 'width: 100%;'}
-        .footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals: {events_url: resolve_url('/events')})
-
-  -# Signup and Highlights
-
-  -# Show sign up form above highlights when we're trying to increase sign ups. Otherwise, feature highlights first which is more engaging and evergreen.
-  -if ["pre-hoc", "soon-hoc"].include?(hoc_mode)
-    = view :home_signup, hoc_mode: hoc_mode
-    = view :home_highlights
-  -else
-    = view :home_highlights
-    = view :home_signup, hoc_mode: hoc_mode
+  %section.map
+    .wrapper
+      - unless @country == 'ma' # Hide map for Morocco because of sensitive border disputes misrepresented in map
+        #maparea.full-width
+          -# Show a picture of the map unless it is currently the week of HOC.
+          -if hoc_mode != "actual-hoc"
+            = view :hoc_events_map
+            .footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals: {events_url: resolve_url('/events')})
+          -else
+            %a{href: resolve_url('/map')}
+              %img{src: '/images/fit-2276/map.jpg', style: 'width: 100%;'}
+            .footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals: {events_url: resolve_url('/events')})
+
+  -# Join the movement
+  %section#join.join-us
+    .wrapper
+      %h2=hoc_s(:hoc_live_join_us)
+      %h3=hoc_s(:hoc_homepage_join_the_movement_description)
+      -if ["pre-hoc", "soon-hoc"].include?(hoc_mode)
+        = view :home_signup, hoc_mode: hoc_mode
+
+  -# What is HoC?
+  -# This section remains evergreen, and the video no longer moves next to the signup form during different hoc_modes.
+  %section.what-is-hoc
+    .wrapper
+      %h2=hoc_s(:front_what_is_hoc)
+      %h3=hoc_s(:hoc_homepage_what_is_hoc_subhead)
+      =view :index_video
+      %p=hoc_s(:hoc_homepage_what_is_hoc_description)
+      %a{href: resolve_url('/learn')}
+        %button=hoc_s(:cta_try_an_activity)
+      //%p=hoc_s(:hoc_homepage_what_is_hoc_cta_how_to_guide, markdown: :inline, locals: {howto_guide_url: resolve_url('/how-to')})
```

Comment:
This is commented out due to a broken link (https://github.com/code-dot-org/code-dot-org/pull/48237) — will add this back once it's fixed on production

Reply:
This is still showing up for me when I pull this branch

![Uploading Screen Shot 2022-09-23 at 8.38.21 AM.png…]()

Reply:
Oh yeah it's gone now 🤷‍♀️